lib/wrappers/openssl.nim

```diff
@@ -342,6 +344,22 @@ proc SSL_CTX_use_PrivateKey_file*(ctx: SslCtx,
 proc SSL_CTX_check_private_key*(ctx: SslCtx): cInt{.cdecl, dynlib: DLLSSLName,
     importc.}
 
+when getEnv("NIM_SSL_CERT_VALIDATION") != "insecure":
```

Comment:
We don't read env vars at compile-time. We use the ``--define`` and Nim's config mechanism instead.